core/src/main/resources/error/error-classes.json

```
@@ -511,6 +511,12 @@
     ],
     "sqlState" : "38000"
   },
+  "FAILED_PARSE_EMPTY_STRING" : {
```

This error class is pretty JSON specific. So, add JSON in the name. For example:
```suggestion
  "EMPTY_JSON_FIELD_VALUE" : {
```